dvc/config.py

```diff
@@ -342,6 +342,7 @@ def edit(self, level="repo"):
         yield conf
 
         conf = self._save_paths(conf, self.files[level])
+        self.validate(conf)
```

Comment:
We need to merge configs before validating. This allows stuff like:
```
dvc remote add  myssh ssh://127.0.0.1
dvc remote modify myssh password mypass --local
```
when password is stored in `.dvc/config.local`, which is not valid by itself, but is valid when merged with `.dvc/config` during normal operation.

Reply:
got it